bridge.h

```diff
@@ -36,6 +36,10 @@ int remove_icc_profile(VipsImage *in);
 
 // Operations
 int resize_image(VipsImage *in, VipsImage **out, double scale, double vscale, int kernel);
+int rot_image(VipsImage *in, VipsImage **out, VipsAngle angle);
+int rot_image90(VipsImage *in, VipsImage **out);
```

Comment:
do we need these other methods versus just the single `rot_image` method where the angle is passed in?